site-content/source/modules/ROOT/pages/development/dependencies.adoc

```diff
@@ -12,13 +12,26 @@ As Cassandra is an Apache product, all included libraries must follow
 Apache's https://www.apache.org/legal/resolved.html[software license
 requirements].
 
+=== Changes to dependency management in Cassandra 4.1
+
+Before Cassandra 4.1, dependencies were managed in `build.xml` and managed with
+Maven Ant Tasks. In Cassandra 4.1 and onwards, dependencies are managed in POM
+templates in `.build/`. New dependencies should be added to the parent POM
```

Comment:
please flesh this out more, such as what are the files to modify. "read the JIRA" isn't good for our docs...